composer.json

```diff
@@ -36,7 +36,7 @@
     "symfony/validator": "^5.1",
     "symfony/yaml": "^5.1@dev",
     "typhonius/acquia-logstream": "dev-master",
-    "typhonius/acquia-php-sdk-v2": "^2.0.14",
+    "typhonius/acquia-php-sdk-v2": "dev-master as 2.0.14",
```

Comment:
Looks like 2.0.15 is out now, I'd prefer using `^2.0.15` since aliases lead to weird dependency conflicts.